magento/pwa-studio
# PB-Videos: Added Page Builder integration videos

## Description

Added long lost Page Builder integration videos to their proper place in the overview page of the PWA Page Builder integration docs.

## Related Issue

Closes #2631 

## Acceptance 

- @omiroshnichenko 

### Verification Stakeholders

- @jcalcaben 

### Verification Steps

1. Build the docs site locally.
2. Verify that the videos load and run.

## Screenshots / Screen Captures (if appropriate)

![image](https://user-images.githubusercontent.com/1828494/90339061-e9a0f000-dfb3-11ea-968b-1d52128056f9.png)

## Checklist

* I have updated the documentation accordingly, if necessary.
